addons/payment/views/payment_portal_templates.xml

```diff
@@ -160,10 +160,6 @@
         <div t-if="payment_tx_id and payment_tx_id.state == 'done'" class="alert alert-success alert-dismissable">
             <button type="button" class="close" data-dismiss="alert" title="Dismiss" aria-label="Dismiss">&amp;times;</button>
             <span t-if='payment_tx_id.acquirer_id.done_msg' t-raw="payment_tx_id.acquirer_id.done_msg"/>
-            <span t-if='thanks_msg' t-raw="thanks_msg"/>
-            <div t-if="thanks_msg and payment_tx_id.acquirer_id.provider == 'transfer' and reference">
```

Comment:
I'd keep all of this for future reference. So the user knows which reference was used for his payment.

Reply:
The message will be like this :

![tx_message](https://user-images.githubusercontent.com/45560557/53237064-13ff3f00-3696-11e9-8c12-473eaef677e3.png)

I can also, remove the message and leave the 'Communication:' part.

Reply:
No that's fine, let's keep it like this for now.